### DISCUSSION TITLE
How to disable "renovate/artifacts" pipeline

### DISCUSSION THREAD

**Discussion by XFNeo** [renovatebot/renovate#12559] [Request Help] [answered]
How to disable "renovate/artifacts" pipeline ?
![image](https://user-images.githubusercontent.com/37923089/140775446-518d215c-aea0-46ee-b6e7-55b85c4cd774.png)
![image](https://user-images.githubusercontent.com/37923089/140775470-88407d9a-0c63-41e1-8b3f-c87fcf42fb65.png)

**Comment by rarkins** [comment, marked as answer]
It's not possible currently. It is set to failed if artifacts uodatibg has failed, such as go.sum. It's not really a "pipeline"

**Reply by XFNeo** [reply]
Yes, I got it. But we have module "k8s.io/client-go@v12.0.0+incompatible" this version has problem with go sum, but we cant update it due to backward compatibility (we excluded this module from renovate bot).
But this microservice successfully builds on our builder and we don`t need failed pipeline from renovate to be able automerge this MR.
```
 WARN: artifactErrors (repository=service1, branch=renovate/module1)
       "artifactErrors": [
         {
           "lockFile": "go.sum",
           "stderr": "Command failed: go get -d ./...
verifying k8s.io/client-go@v12.0.0+incompatible/go.mod: checksum mismatch
	downloaded: h1:7vJpHMYJwNQCWgzmNV+VYUl1zCObLyodBc8nIyt8L5s=
	go.sum:     h1:E95RaSlHr79aHaX0aGSwcPNfygDiPKOVXdmivCIZT0k=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'."
         }
       ]
```

**Comment by XFNeo** [comment]
@rarkins Any news?)

**Reply by rarkins** [reply]
I marked my previous response as the answer, so it's clear: there is no way to disable it

**Comment by vistasunil** [comment]
@rarkins How we do this in Air gapped environment then? There must be a way to disable renovate looking to internet for these artifacts?

**Reply by RahulGautamSingh** [reply]
If you want to stop artifacts update, use https://docs.renovatebot.com/configuration-options/#skipartifactsupdate

**Comment by vistasunil** [comment]
@RahulGautamSingh thanks for sharing this option.. I will test it. But can you pls also suggest, how we can do it from renovate config file i.e. renovate.json? ..is it possible?

**Reply by RahulGautamSingh** [reply]
Yes, you add that option in the renovate config file :)

**Comment by vistasunil** [comment]
@RahulGautamSingh I tried this but it also did not help. PR still gets additions build renovate/artifacts and it fails.

**Reply by RahulGautamSingh** [reply]
Please create a minimal reproduction and I will look into it

**Comment by vistasunil** [comment]
@RahulGautamSingh We fixed it by whitelisting GOSUMDB url itself in our proxy. You can easily validate the setting you shared by running renovate with it on any repo having GO dependencies. You will see that it doesn't work.
